src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,54 @@
+package view;
+
+import java.util.List;
+import model.car.Car;
+
+public class OutputView {
+    private static final OutputView instance = new OutputView();
+
+    public static OutputView getInstance() {
+        return instance;
+    }
+
+    private OutputView() {
+    }
+
+    public void printExceptionMessage(Exception exception) {
+        System.out.println(exception.getMessage());
+    }
+
+    public void printResultMessage() {
+        System.out.println(Message.OUTPUT_RESULT_MESSAGE.message);
+    }
+
+    public void printResult(List<Car> cars) {
+        for (Car car : cars) {
+            System.out.printf(Message.OUTPUT_RESULT_FORMAT.message, car.getName(), createResultDisplay(car));
+        }
+        System.out.println();
+    }
+
+    private static StringBuilder createResultDisplay(Car car) {
+        StringBuilder result = new StringBuilder();
+        for (int i = 0; i < car.getPosition(); i++) {
+            result.append("-");
+        }
+        return result;
```

Comment:
java 11을 사용하고 계시다면 String의 repeat 함수를 활용할 수 있을 것 같아요!
```suggestion
        return "-".repeat(car.getPosition());
```